stdlib/ssl.pyi

```diff
@@ -4,30 +4,48 @@ import sys
 from _typeshed import ReadableBuffer, StrOrBytesPath, WriteableBuffer
 from collections.abc import Callable, Iterable
 from typing import Any, NamedTuple, overload
-from typing_extensions import Literal, Never, Self, TypeAlias, TypedDict, final
+from typing_extensions import Literal, Never, Self, TypeAlias
+
+from _ssl import (
```

Comment:
Could you add `"_ssl"` to the table here, so that ruff puts the import in the right place?

https://github.com/python/typeshed/blob/3171cb44aa9d7c7ada31ef78720b42ea28acd24a/pyproject.toml#L50-L93